opm/autodiff/FlowMainEbos.hpp

```diff
@@ -546,8 +543,7 @@ namespace Opm
             {
                 const EclipseGrid& inputGrid = eclState().getInputGrid();
                 eclipse_writer_.reset(new EclipseWriter(eclState(), UgGridHelpers::createEclipseGrid( grid , inputGrid )));
-                eclipse_writer_->writeInitial(geoprops_->simProps(grid),
-                                              geoprops_->nonCartesianConnections());
+                eclipse_writer_->writeInitial(computeLegacySimProps_(), eclState().getInputNNC());
```

Comment:
We should make sure that "non-geometric" NNCs *are* handled by flow_ebos. I am not sure what would be the best way to do it, though. These connections should also be dealt with in load-balancing etc., so perhaps we should make them part of the actual grid (CpGrid)?